src/service/userService.js

```diff
@@ -5,12 +5,12 @@ import { addUserAsync } from "../dataAccess/userDataAccess";
  * @param {Object} userInformation The information related to the user.
  * @returns The user information saved.
  */
-export const addNewUser = async (userInformation) => {
+export const addNewUserAsync = async (userInformation) => {
     const newUser = {
-        name: userInformation.name,
-        lastName: userInformation.lastName,
-        email: userInformation.email,
-        password: userInformation.password
+        Name: userInformation.name,
+        LastName: userInformation.lastName,
+        Email: userInformation.email,
+        Password: userInformation.password
```

Comment:
You will hate me jeje, but please move this logic into the data access file. It is because the service should not be in charge about how data base is the formatted. It should be the data access the responsible about how they want to save the data